public/resources/js/form.js

```diff
@@ -1,76 +1,89 @@
-    var pass = document.getElementById("password");
-    var msg = document.getElementById("message");
-    var str = document.getElementById("strenght");
+var icon = document.getElementsByClassName("icon");
+var pass = document.getElementById("password");
+var msg = document.getElementById("message");
+var str = document.getElementById("strenght");
 
-    var copass = document.getElementById("cpassword");
-    var nmsg = document.getElementById("nomessage");
-    var nostr = document.getElementById("constrength");
+var copass = document.getElementById("cpassword");
+var nmsg = document.getElementById("nomessage");
+var nostr = document.getElementById("constrength");
 
-    var email = document.getElementById("email");
-    var emsg = document.getElementById("emailmsg");
-    var estr = document.getElementById("emailstrenght");
+var email = document.getElementById("email");
+var emsg = document.getElementById("emailmsg");
+var estr = document.getElementById("emailstrenght");
 
-    pass.addEventListener('input',() => {
-        if(pass.value.length > 0){
-            msg.style.display = "block";
-        }else{
-            msg.style.display = "none";
+pass.addEventListener('input',() => {
+    if(pass.value.length > 0){
+        msg.style.display = "block";
+        for (let i = 0; i < icon.length; i++) {
+            icon[i].style.top = "47px";
         }
-        if(pass.value.length < 4){
-            str.innerHTML = "weak";
-            pass.style.borderColor = "#b22323";
-            msg.style.color = "#b22323";
-        } else if(pass.value.length >= 4 && pass.value.length < 8){
-            str.innerHTML = "medium";
-            pass.style.borderColor = "#bc5806";
-            msg.style.color = "#bc5806";
-        } else if(pass.value.length >= 8){
-            str.innerHTML = "strong";
-            pass.style.borderColor = "#356212";
-            msg.style.color = "#356212";
-        }
-        checkPasswordConfirmation();
-    });
+        
+    }else{
+        msg.style.display = "none";
+        pass.style.backgroundColor = "rgba(255,255,255,0.2)";
+    }
+    if(pass.value.length > 0 && pass.value.length < 4){
+        str.innerHTML = "weak";
+        pass.style.backgroundColor = "rgba(211, 12, 12, 0.486)";
+        msg.style.color = "#b11010";
+        msg.style.marginBottom = "10px";
+        msg.style.top = "-8px";
+        msg.style.position = "relative";

```

Comment:
instead of repeting the same in each if statement you could make a sass class like
```
password-strenght {
  &__weak {
   &::after {
     content: 'weak'
   }
   bg-color: rgba(...
   color: #...
   margin: ...
   top:,,,,
  }
  &__medium {
  ....
  }
  &__strong {
  ...
  }
}
```
and make something like this in the js
```
class = 'password-strenght'
if password {
  element.addClass(class)
}
if weak {
 class = class,split('__').shift
 class += '__weak'
} else if medium {
 class = class,split('__').shift
 class += '__medium'
} else if string {
 class = class,split('__').shift
 class += '__weak'
}
```
something like this, will reduce the code a lot and ensure the good practices not making styles in a not style compiler(sass, csss, etc)